components/maplibre/ban-map/layers.js

```diff
@@ -39,6 +40,72 @@ const TOPONYME_MAX = NUMEROS_MIN + 2
 const TOPONYME_COLOR = '#7c5050'
 export const PARCELLES_MINZOOM = 14
 
+const getColors = () => {
+  const array = []
+  Object.keys(positionsColors).forEach(key => {
+    array.push(key, positionsColors[key].color)
+  })
+
+  return [...array]
+}
+
+const colors = getColors()
+
+export const positionsCircleLayer = {
+  id: 'positions',
+  source: 'positions',
+  type: 'circle',
+  paint: {
+    'circle-color': ['match', ['get', 'type'], ...colors, '#000'],
+    'circle-stroke-color': [
+      'case',
+      ['boolean', ['feature-state', 'hover'], false],
+      theme.primary,
+      '#fff'
+    ],
+    'circle-stroke-width': [
+      'case',
+      ['boolean', ['feature-state', 'hover'], false],
+      3,
+      1
+    ],
+    'circle-radius': {
+      stops: [
+        [12, 0.8],
+        [17, 6]
+      ]
+    }
+  }
+}
+
+export const positionsLabelLayer = {
+  id: 'positions-label',
+  source: 'positions',
+  type: 'symbol',
+  minzoom: NUMEROS_MIN,
+  paint: {
+    'text-color': ['match', ['get', 'type'], ...colors, '#000']
+  },
+  layout: {
+    'text-font': ['Noto Sans Bold'],
+    'text-size': {
+      stops: [
+        [NUMEROS_MIN, 13],
+        [19, 16]
+      ]
+    },
+    'text-field': [
+      'format',
+      ['get', 'type'],
+      ' - ',
+      ['get', 'numero'],
```

Comment:
Pour garder une cohérence avec l'affichage de Mes Adresses.
```suggestion
      ['get', 'numero'],
      ' - ',
      ['get', 'type'],
```